drop-in/src/main/java/com/adyen/checkout/dropin/DropInConfiguration.kt

```diff
@@ -71,9 +75,17 @@ class DropInConfiguration private constructor(
 ) : Configuration {
 
     internal fun <T : Configuration> getConfigurationForPaymentMethod(paymentMethod: String): T? {
-        if (availablePaymentConfigs.containsKey(paymentMethod)) {
+        // For instant payment we cannot use the payment method as key as we don't have a definitive list of supported
+        // instant payment methods.
+        val key = if (InstantPaymentComponent.PROVIDER.isPaymentMethodSupported(paymentMethod)) {
```

Comment:
Let me know if you have a better idea on how to do this

Reply:
Yep makes sense